### ISSUE TITLE
iOS VoiceOver (text-to-voice) doesn't work correctly with the date filter

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#119]
## Description of issue

When VoiceOver is enabled on iOS, attempting to interact with the date filter calendar doesn't go well. Tapping anywhere in the filter UI results in it reading off the entire contents - the Title label (either 'select dates' or '<first date> - <last date>'), "Clear", "<Month> <Year>", "Sun Mon Tue, Wed, Thu, Fri, Sat", followed by every number in the calendar, then "No events".

You end up being trapped in the filter as you can't tap the CTA button as this just re-triggers the read out of the filter :/ 

## Expected behaviour

It should be possible to tap the "Clear" and CTA (show x events) buttons, the change month arrows, and individual days and have them read out individually. It should then be possible to interact with them by double-tapping on them. 
Additionally it should be possible to tap on the selected date range and have that read out. 

## Steps to reproduce

- Enable VoiceOver (Settings > General > Accessibility > VoiceOver)
- Go to the events page in the Pride App
- Open the date filter
- Try to interact with it
> You will find yourself in voiceover hell

The only way to get out of the filter at this point (that I found) was to disable VoiceOver.

## OS / Browser / Device model

iPhone 7+ iOS 11.0

**Comment by declanslevin** [comment]
This issue has been resolved with the latest round of Date filter work
